third_party/move/move-compiler-v2/transactional-tests/tests/tests.rs

```diff
@@ -28,6 +28,11 @@ struct TestConfig {
     runner: fn(&Path) -> datatest_stable::Result<()>,
     experiments: &'static [(&'static str, bool)],
     language_version: LanguageVersion,
+    /// Path substrings for tests to include. If empty, all tests are included.
+    include: &'static [&'static str],
```

Comment:
Shouldn't the outer `'static` imply the inner one?  I don't see how you can have `&'a [&'b str]` where `'b` does not contain `'a`.

Reply:
Compiler error if you omit inner lifetime. Hence, keeping it.